src/main/flight/mixer_tricopter.c

```diff
@@ -95,7 +95,7 @@
 #define TRI_YAW_FORCE_PRECISION     (1000)
 
 #define IsDelayElapsed_us(timestamp_us, delay_us) ((uint32_t)(micros() - timestamp_us) >= delay_us)
-#define IsDelayElapsed_ms(timestamp_ms, delay_ms) ((uint32_t)(millis() - timestamp_ms) >= delay_ms)
+#define IsDelayElapsed_ms(timestamp_ms, delay_ms) ((uint32_t)(now - timestamp_ms) >= delay_ms)
```

Comment:
You should think about this one more time.
It is good practice to check the hw clock only one time per function call.
(Same for any hw read. Even better to only read one time per loop but that might require module globals which is not so nice.)
Enforcing a good practice is not bad.
And it also reduce the code size. No huge difference but anyway; Smaller means quicker.
Maybe a comment?